pkg/containerutil/containerutil.go

```diff
@@ -129,3 +133,121 @@ func WithBindMountHostProcfs(_ context.Context, _ oci.Client, _ *containers.Cont
 	s.Linux.ReadonlyPaths = newROP
 	return nil
 }
+
+func Stop(ctx context.Context, container containerd.Container, timeout *time.Duration) error {
```

Comment:
Can you add some comments here, something like 

```
// Stop stops `container` by sending SIGTERM. If the container is not stopped after `timeout`, it sends a SIGKILL.
```

Reply:
Good point, thanks. Updated.